doc/port_auto_neg/port-auto-negotiation-design.md

```diff
@@ -511,6 +538,52 @@ else if autoneg == false:
         setInterfaceType(port, interface_type)
 ```
 
+##### Getting Negotiated Results
```

Comment:
Please do not add coding snippets to HLD. HLD is to be used by non programmers as well. For explaining the flows please use flow diagrams

Reply:
Sure, will do